boards/arm/samv7/common/Kconfig

```diff
@@ -0,0 +1,96 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfiglanguage.txt in the NuttX tools repository.
+#
+
+config BOARD_SAMV7_COMMON
+	bool "Board common logic"
+	default y
+	help
+		 Board common logic located in each board/common folder.
+
+if BOARD_SAMV7_COMMON
+
+config SAMV7_PROGMEM_OTA_PARTITION
+	bool
+	default n
+	select ARCH_RAMFUNCS
+	select BCH
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_CONFIG
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+	select SAMV7_PROGMEM
+	select SAMV7_PROGMEM_ERASESTATE
+
+config SAMV7_MCUBOOT_HEADER_SIZE
+	hex
+	default 0x200
+	depends on SAMV7_FORMAT_MCUBOOT
+
+menuconfig SAMV7_FORMAT_MCUBOOT
+	bool "MCUboot bootable format"
+	default n
+	select SAMV7_PROGMEM_OTA_PARTITION
+	help
+		 The MCUboot support of loading the firmware images.
+
+if SAMV7_FORMAT_MCUBOOT
+
+config SAMV7_MCUBOOT_BOOTLOADER
```

Comment:
Actually I reread your previous comments and agree that `SAMV7_MCUBOOT_BOOTLOADER` option is not needed at all and everything can be controlled using `BOARDCTL_BOOT_IMAGE` that is selected by the `MCUBOOT_BOOTLOADER` option from the `app` layer.
Sorry for confusion. `SAMV7_MCUBOOT_BOOTLOADER` can be safely removed from this PR